clar2wasm/src/standard/standard.wat

```diff
@@ -1727,6 +1730,46 @@
         )
     )
 
+    ;;
+    ;; logical not implementation
+    ;;
+    (func $not (type $not-func) (param $bool_in i32) (result i32)
+        (i32.eqz (local.get $bool_in))
+    )
+
+    ;;
+    ;; xor implementation
+    ;;
+    (func $xor (type $xor-func) (param $a_lo i64) (param $a_hi i64) (param $b_lo i64) (param $b_hi i64) (result i64 i64)
+        (local.get $a_lo)
+        (local.get $a_hi)
+        (local.get $b_lo)
+        (local.get $b_hi)
+        (call $bit-xor)
+    )
```

Comment:
if `$xor` is the exact same as `$bit-xor`, is it worth creating a new function?
At worst, we could even just `(export "xor" (func $bit-xor))`.

Reply:
Agreed. Let's delete this and just call `bit-xor` directly. But speaking of calling `bit-xor`... where is `xor` being handled? I don't see a Word added for it, but your new tests seem to be passing... what am I missing?

Reply:
I just debugged to see what was happening... It's hitting the else and calling `xor` from the standard library as a user-defined function! @csgui please add a `Word` for `xor` in _bitwise.rs_ so that it follows the right codepath here. We need to solve #86 so that these tests would properly trigger an error.

Reply:
I've added a `Word` for `xor`.